x/paramfilter/test/governance_params_test.go

```diff
@@ -0,0 +1,161 @@
+package test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/celestiaorg/celestia-app/x/paramfilter"
+	"github.com/stretchr/testify/suite"
+
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+
+	testutil "github.com/celestiaorg/celestia-app/test/util"
+	minttypes "github.com/celestiaorg/celestia-app/x/mint/types"
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
+	params "github.com/cosmos/cosmos-sdk/x/params"
+	"github.com/cosmos/cosmos-sdk/x/params/types/proposal"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+)
+
+type HandlerTestSuite struct {
+	suite.Suite
+
+	app        *app.App
+	ctx        sdk.Context
+	govHandler v1beta1.Handler
+	pph        paramfilter.ParamBlockList
+}
+
+func (suite *HandlerTestSuite) SetupTest() {
+	suite.app, _ = testutil.SetupTestAppWithGenesisValSet(app.DefaultConsensusParams())
+	suite.ctx = suite.app.BaseApp.NewContext(false, tmproto.Header{})
+	suite.govHandler = params.NewParamChangeProposalHandler(suite.app.ParamsKeeper)
+	suite.pph = paramfilter.NewParamBlockList([2]string{})
+
+	minter := minttypes.DefaultMinter()
+	suite.app.MintKeeper.SetMinter(suite.ctx, minter)
+}
+
+func TestHandlerTestSuite(t *testing.T) {
+	suite.Run(t, new(HandlerTestSuite))
+}
+
+func (suite *HandlerTestSuite) TestUnmodifiableParameters() {
+	testCases := []struct {
+		name     string
+		proposal *proposal.ParameterChangeProposal
+		onHandle func()
+		expErr   bool
+	}{
+		// The tests below show the parameters as modifiable, block in App.BlockedParams().
+		{
+			"bank.SendEnabled",
+			testProposal(proposal.ParamChange{
+				Subspace: banktypes.ModuleName,
+				Key:      string(banktypes.KeySendEnabled),
+				Value:    `[{"denom": "test", "enabled": false}]`,
+			}),
+			func() {
+				sendEnabledParams := suite.app.BankKeeper.GetParams(suite.ctx).SendEnabled
+				suite.Require().Equal([]*banktypes.SendEnabled{banktypes.NewSendEnabled("test", false)}, sendEnabledParams)
```

Comment:
[optional] consider extracting variables to make the test more readable:

```suggestion
				got := suite.app.BankKeeper.GetParams(suite.ctx).SendEnabled
				want := []*banktypes.SendEnabled{banktypes.NewSendEnabled("test", false)}
				suite.Require().Equal(want, got)
```